staging/src/k8s.io/apiextensions-apiserver/test/integration/helpers.go

```diff
@@ -92,3 +95,17 @@ func updateCustomResourceDefinitionWithRetry(client clientset.Interface, name st
 	}
 	return nil, fmt.Errorf("too many retries after conflicts updating CustomResourceDefinition %q", name)
 }
+
+// GetCRDColumnsForVersion returns the columns for given version in given CRD Spec.
+func GetCRDColumnsForVersion(crdSpec *apiextensionsv1beta1.CustomResourceDefinitionSpec, version string) ([]apiextensionsv1beta1.CustomResourceColumnDefinition, error) {
+	if !utilfeature.DefaultFeatureGate.Enabled(apiextensionsfeatures.CustomResourceWebhookConversion) ||
```

Comment:
We are making feature gates non-global, but pass them through in the apiserver bootstrapping. Integation tests and helpers depending on the global gates are the wrong direction. 

Again the rule of thumb: make helpers independent from the feature gates, and move the depending logic to the actual logic (of tests, of registries/strategies, etc.).